src/components/Blog/BlogCarousel/CarouselContainer.style.ts

```diff
@@ -11,22 +11,24 @@ export interface CarouselContainerBoxProps extends Omit<BoxProps, 'variant'> {
   styles?: CSSObject;
 }
 
-export const CarouselContainerBox = styled(Box)<CarouselContainerBoxProps>(
-  ({ theme }) => ({
-    display: 'flex',
-    gap: theme.spacing(4),
-    marginTop: theme.spacing(3),
-    overflow: 'auto',
-    width: '100%',
-    overflowY: 'hidden',
-    scrollSnapType: 'x mandatory',
-    '& > *': {
-      flexShrink: 0,
-      scrollSnapAlign: 'center',
-    },
-    '::-webkit-scrollbar': { display: 'none' },
-  }),
-);
+export const CarouselContainerBox = styled(Box, {
+  shouldForwardProp: (prop) => prop !== 'styles',
+})<CarouselContainerBoxProps>(({ theme, styles }) => ({
+  display: 'flex',
+  gap: theme.spacing(4),
+  marginTop: theme.spacing(3),
+  overflow: 'auto',
+  width: '100%',
+  overflowY: 'hidden',
+  scrollSnapType: 'x mandatory',
+  paddingBottom: theme.spacing(0.75),
+  '& > *': {
+    flexShrink: 0,
+    scrollSnapAlign: 'center',
+  },
+  '::-webkit-scrollbar': { display: 'none' },
+  ...styles,
```

Comment:
Why passing styles there? You should be able to use like other styles across the app